head.html

```diff
@@ -1,4 +0,0 @@
-<meta name="viewport" content="width=device-width, initial-scale=1"/>
```

Comment:
Do we intend to remove this? It appears to break the page styles when doing so

Reply:
Oh that was my bad, I thought it wasn't in use because the head was being loaded from the drive. I'll add it back, one sec

Reply:
Ok should be good now

Reply:
thanks!